### ISSUE TITLE
Amiwrap.m: File-existance test necessary?

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/AMICI#32]
Is the following check in amiwrap.m necessary?

```
    if(exist(symfun,'file') ~= 2)
        error(['"' symfun '" must be the name of a matlab function in the matlab path. Please check whether the folder containing "' symfun '" is in the matlab path. AMICI currently does not support absolute or relative paths in its input arguments.'])
    end
```
Amiwrap expects a function name, but checks if the file exists. I think this is an unnecessary limitation.

For example, it limits the use of package functions such as:
''' 
amiwrap('bla', 'package.bla', pwd);
'''

I suggest replacing it by try/catch.

**Comment by FFroehlich** [comment]
that's correct, it fact it should already be possible to pass arbitrary functions and structs to amiwrap.

**Comment by FFroehlich** [comment]
fixed in 714ad6c
